src/header.tsx

```diff
@@ -88,6 +90,17 @@ export const filterColumns = [
             label: _("Last modified"),
         },
     },
+    {
+        title: _("Owner"),
+        [SortByDirection.asc]: {
+            itemId: Sort.owner_asc,
+            label: _("Owner ascending"),
```

Comment:
@jelly oh yeah Garrett never commented on this and the `Owner descending` string. I am open for a discussion if you'd like to change it :)